db/database.js

```diff
@@ -70,23 +79,39 @@ const createEnrollmentsTable = db => {
     `CREATE TABLE enrollments (
       id INTEGER PRIMARY KEY AUTOINCREMENT,
       cohort_id VARCHAR,
-      student_id VARCHAR NOT NULL UNIQUE,
+      student_id VARCHAR NOT NULL,
       enrollment_status VARCHAR,
       CONSTRAINT fk_cohort_id
         FOREIGN KEY (cohort_id)
         REFERENCES cohorts(id),
       CONSTRAINT fk_student_id
         FOREIGN KEY (student_id)
-        REFERENCES students(id),
-      CONSTRAINT fk_enrollment_status
-        FOREIGN KEY (enrollment_status)
-        REFERENCES enrollment_status(id)
+        REFERENCES students(id)
     )`,
     err => {
       if (err) {
-        // Table already created
+        console.log('error', err);
       } else {
-        // create rows
+        const insert = 'INSERT INTO enrollments VALUES (?, ?, ?, ?)';
+        studentsList.forEach(student => {
+          const sql = `select id, status, cohort_id from students where first_name=? and last_name=?`;
+          db.get(sql, [student[2], student[3]], (error, rows) => {
+            if (error) {
+              throw error;
+            }
+            const studentId = rows.id;
+            const { status } = rows;
+            const cohortId = rows.cohort_id;
+            const cohortSql = `select id from cohorts where cohort_id=?`;
+            db.get(cohortSql, cohortId, (error1, row) => {
+              if (error1) {
+                throw error1;
+              }
+              const { id } = row;
+              db.run(insert, [null, id, studentId, status]);
+            });
+          });
+        });
```

Comment:
Will refactor to add helper methods and api in next pr once we confirm the schema looks ok